core/src/main/java/com/hcl/ecomm/core/config/DeleteCartItemServiceConfig.java

```diff
@@ -0,0 +1,21 @@
+package com.hcl.ecomm.core.config;
+
+import org.osgi.service.metatype.annotations.AttributeDefinition;
```

Comment:
@nitheshkumar-hcl : why do we need a separate config for delete , you can reuse existing cofigs for product and login

Reply:
Removed this Config